core/src/themes/ionic/ionic.theme.default.scss

```diff
@@ -10,85 +10,211 @@
 // Default Ionic Colors
 // -------------------------------------------------------------------------------------------
 // Color map should provide
-// - base: main color to be used.
-// - contrast: Color that will provide readable text against the base color
-// - shade: 12% darker version of the base color (mix with black)
-// - tint: 10% lighter version of the base color (mix with white)
+// - bold: a map of the bold color variations
+// - subtle: a map of the subtle color variations
+//
+// Each hue color map should provide
+// TODO: rename base to background?
+// - base: The main color used for backgrounds
+// - base-rgb: The base color in RGB format
+// - contrast: A color that ensures readable text on the base color
+// - contrast-rgb: The contrast color in RGB format
+// - shade: A darker variant of the base color, used for pressed/active states
+// - tint: A lighter variant of the base color, used for ?
+// - foreground: The main color used for text and foreground elements
 
 // TODO(ROU-10778, ROU-10875): Sync the color names to the design system of
 // ios and md. This will allow us to have a single color map.
-$primary: #105cef;
-$secondary: initial;
-$tertiary: initial;
-$success: #1fbd3b;
-$warning: #e18300;
-$danger: #bf2222;
-$light: #f2f4fd;
-$medium: initial;
-$neutral: #626262;
-$dark: initial;
-
 $ionic-colors: (
   primary: (
-    base: $primary,
-    contrast: #fff,
-    shade: color.get-color-shade($primary),
-    tint: color.get-color-tint($primary),
-  ),
-  secondary: (
-    base: $secondary,
-    contrast: $secondary,
-    shade: color.get-color-shade($secondary),
-    tint: color.get-color-tint($secondary),
-  ),
-  tertiary: (
-    base: $tertiary,
-    contrast: $tertiary,
-    shade: color.get-color-shade($tertiary),
-    tint: color.get-color-tint($tertiary),
-  ),
-  success: (
-    base: $success,
-    contrast: #000,
-    shade: color.get-color-shade($success),
-    tint: color.get-color-tint($success),
+    bold: (
+      base: globals.$ion-bg-primary-base-default,
+      base-rgb: globals.$ion-bg-primary-base-default-rgb,
+      contrast: globals.$ion-text-inverse,
+      contrast-rgb: globals.$ion-text-inverse-rgb,
+      shade: globals.$ion-bg-primary-base-press,
+      // TODO this is not a verified value
+      tint: globals.$ion-semantics-primary-600,
```

Comment:
I used the token one step lighter than the base for all tint values

Reply:
Sounds good to me for now, the Ionic Design System doesn't have these exact concepts (at least from UX Team), so this does the job I think